common/src/main/resources/data/sql/migration/V1-V2/mysql.sql

```diff
@@ -0,0 +1,13 @@
+-- MySQL Conversion from V1 -> V2
```

Comment:
I'd like the path of these files to stay consistent with the enum name of the version that triggers them, so this one would be `.../migration/V001/mysql.sql`